subprojects/docs/src/docs/userguide/thirdPartyIntegration.adoc

```diff
@@ -0,0 +1,48 @@
+// Copyright 2018 the original author or authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+[[third_party_integration]]
+== Gradle & Third-party Tools
+
+Gradle can be integrated with many different third-party tools such as IDEs and continuous integration platforms. Here we look at some of the more common ones as well as how to integrate your own tool with Gradle.
+
+=== IDEs
```

Comment:
❓ do you think Visual Studio Code or Android Studio deserve separate mentions here?

Reply:
Only if we can usefully say anything about them. Android Studio can at least be mentioned with a link or reference to IntelliJ IDEA. I assume the IDEA plugin works in the same way for both.

Is there any integration with Visual Studio Code?

Reply:
👍